pontoon/base/models.py

```diff
@@ -2245,6 +2246,12 @@ def reset_active_translations(self, locale):
 
         translations.filter(pk__in=unreviewed_pks).update(active=True)
 
+    def get_or_create(self, defaults=None, **kwargs):
+        return super(EntityQuerySet, self).get_or_create(defaults, **kwargs)
+
+    def bulk_update(self, objs, fields=None, batch_size=None):
```

Comment:
Hey @Karskaya,
That's exactly what I've meant. The only thing that's missing for me is a small test to check if the method works as we expect.

Additionally, It could be a good idea to wrap this PendingDeprecationWarning and throw a warning if the version of Django is equal to or greater than 2.

What do you think about it? (cc: @mathjazz)

Reply:
Sounds good to me!

Reply:
Hello @jotes!
Could you please explain one more time this part with PendingDeprecationWarning?

Did you mean something like this?
```
def bulk_update(self, objs, fields=None, batch_size=None):
        if django.get_version() >= LooseVersion("2"):
            msg = "Warning message about Django version"
            warnings.warn(msg, PendingDeprecationWarning)
        return bulk_update(objs, fields, batch_size)
```

cc: @mathjazz

Reply:
Something like that, year. Or just add a reference to [bug 1600344](https://github.com/mozilla/pontoon/blob/1c90890/pontoon/base/middleware.py#L23).

Reply:
@mathjazz Please, review again.
I've remove all word_count related code for now and changed version check.
To make this PR finished I need a test, according to 
  
> The only thing that's missing for me is a small test to check if the method works as we expect.

Any advice with test? Should it looks the same as tests in test_entity.py? How can I create `objs` data? Can it be done like this? - https://github.com/mozilla/pontoon/blob/3462eff11975cf5442bc03d3ba6b617666f94fd2/pontoon/base/tests/managers/test_entity.py#L63-L71



>